reference/glossary.md

```diff
@@ -0,0 +1,445 @@
+
+See https://discourse.ubuntu.com/t/about-application-streaming/17769.
+
+<a name="snap"></a>
+### Snap
+
+A software package for a desktop, cloud or IoT application that is easy to install, secure, cross‐platform and dependency‐free.
+
+See [the Snapcraft website](https://snapcraft.io/).
+
+<a name="swrast"></a>
+### Software Rasterization (swrast)
+
+An LLVMpipe-based software rendering platform that is useful for visual tests but does not provide audio input/output.
+
+See https://discourse.ubuntu.com/t/anbox-platforms/18733.
+
+<a name="stream-agent"></a>
+### Stream agent
+
+The software running on a server equipped with a GPU connected to Anbox Cloud, which serves as an entry point that the stream gateway can connect to.
+
+See https://discourse.ubuntu.com/t/about-application-streaming/17769.
+
+<a name="stream-gateway"></a>
+### Stream gateway
+
+The central component that connects clients with stream agents. Its role is to choose the best possible region depending on the user location and server capacities.
+
+See https://discourse.ubuntu.com/t/about-application-streaming/17769.
+
+<a name="streaming-stack"></a>
+### Streaming stack
+
+A collection of components designed to run containers on GPU-equipped machines and stream their visual output to clients via WebRTC.
+
+See https://discourse.ubuntu.com/t/about-application-streaming/17769.
+
+<a name="stun/turn-server"></a>
+### STUN/TURN server
+
+A server that finds the most optimal network path between a client and the container running its application.
+
+<a name="ubuntu-advantage-for-applications"></a>
+### Ubuntu Advantage for Applications
+
+Canonical’s service package for Ubuntu that provides enterprise security and support for open-source applications, with managed service offerings available. Note the difference between Ubuntu Advantage for Infrastructure and Ubuntu Advantage for Applications; Anbox Cloud requires a Ubuntu Advantage for Applications subscription.
+
+See [Ubuntu Advantage](https://ubuntu.com/support).
+
+<a name="ubuntu-one"></a>
+### Ubuntu One
+
+A central user account system used by all Canonical sites and services. You need a Ubuntu One account to purchase the Ubuntu Advantage for Applications subscription that is required to run Anbox Cloud, and to log into the web dashboard.
+
+See [Ubuntu One](https://login.ubuntu.com/).
+
+<a name="watchdog"></a>
+### Watchdog
+
+A software component that monitors the app in a container and terminates the container if the app crashes or is moved to the background.
+
+See [Watchdog settings](https://discourse.ubuntu.com/t/application-manifest/24197#watchdog).
+
+<a name="web-dashboard"></a>
+### Web dashboard
+
+A web GUI for Anbox Cloud from where developers can create, manage and stream applications from their web browser. Previously called Dev UI.
```

Comment:
Is "previously called" okay, or do we need to state that it is a replacement?

Reply:
I'm even wondering if it's worth mentioning the Dev UI at all. It was phased out quite a few releases ago
Technically speaking though, it's a full blown out replacement, it's a different name, code, deployment method, and feature set

Reply:
"Dev UI" still occurs in the docs, but maybe it would be better to get rid of it there than to mention it here?

Reply:
I vote to get rid of it in the docs. I think it would mostly create confusion, what do you think @adglkh @morphis ?

Reply:
Yes, lets get rid of "Dev UI" as term.